### PR TITLE
test(components/lists): add missing accessibility test to repeater

### PR DESCRIPTION
**Pull request by Blackbaud-CoreyArcher**
[AB#2340607](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/2340607)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/1252/storybook/)

Component Storybooks:

- [a11y-storybook](https://blackbaud.github.io/skyux-pr-preview/1252/storybooks/a11y-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/1252/storybooks/ag-grid-storybook/)
- [core-storybook](https://blackbaud.github.io/skyux-pr-preview/1252/storybooks/core-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/1252/storybooks/indicators-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/1252/storybooks/layout-storybook/)
- [lists-storybook](https://blackbaud.github.io/skyux-pr-preview/1252/storybooks/lists-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/1252/storybooks/modals-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/1252/storybooks/pages-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/1252/storybooks/popovers-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/1252/storybooks/split-view-storybook/)
- [storybook](https://blackbaud.github.io/skyux-pr-preview/1252/storybooks/storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/1252/storybooks/tabs-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/1252/storybooks/theme-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/1252/storybooks/tiles-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/1252/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/1252/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/1252/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/1252/playground/)

File: libs/components/lists/src/lib/modules/repeater/repeater.component.spec.ts [hunk 1]
```diff
@@ -1212,6 +1212,19 @@ describe('Repeater item component', () => {
 
       flushDropdownTimer();
     }));
+
+    it('should be accessible', async () => {
+      cmp.showRepeaterWithActiveIndex = true;
+      cmp.expandMode = 'none';
+      cmp.activeIndex = 0;
+      // Detect active index.
```
Comment (Blackbaud-PaulCrowder) [inline libs/components/lists/src/lib/modules/repeater/repeater.component.spec.ts:1224]: Put some empty lines between these logical blocks for readability.

```
it('should be accessible', async () => {
      cmp.showRepeaterWithActiveIndex = true;
      cmp.expandMode = 'none';
      cmp.activeIndex = 0;

      // Detect active index.
      fixture.detectChanges();
      await fixture.whenStable();

      // Role changes on next cycle.
      fixture.detectChanges();
      await fixture.whenStable();

      await expectAsync(fixture.nativeElement).toBeAccessible();
```

Comment (Blackbaud-CoreyArcher) [reply libs/components/lists/src/lib/modules/repeater/repeater.component.spec.ts:1224]: updated

**Review by Blackbaud-PaulCrowder — DISMISSED** [review]

**Review by Blackbaud-TrevorBurch — APPROVED** [review]

**Comment by blackbaud-sky-build-user** [comment]
Cherry-pick successful! :cherries: :tada: See https://github.com/blackbaud/skyux/pull/1277

**Pull request merged**